ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/mainnet/MainnetProtocolSpecs.java

```diff
@@ -193,6 +193,12 @@ private MainnetProtocolSpecs() {}
         .name("Constantinople");
   }
 
+  public static ProtocolSpecBuilder<Void> petersburgDefinition(final int chainId) {
+    return constantinopleDefinition(chainId)
+        .gasCalculator(TangerineWhistleGasCalculator::new)
```

Comment:
Good spot - we will because Constantinople gas calculator also provides the gas cost for EXTCODEHASH and CREATE2 which aren't available in tangerine whistle. It should omit the `calculateStorageCost` and `calculateStorageRefundAmount` overrides that are in `ConstantinopleGasCalculator`.